MIGRATE.md

```diff
@@ -1 +1,161 @@
+## webpack-migrate
 
+The `migrate` feature eases the transition from [version 1](http://webpack.github.io/docs/) to [version 2](https://gist.github.com/sokra/27b24881210b56bbaff7). `migrate` 
+also allows users to switch to the new version of webpack without having to extensively [refactor](https://webpack.js.org/guides/migrating/).
+
+### Usage
+To use `migrate`, run the following command, with the value of `<config>` being a path to an existing webpack configuration file
+
+```bash
+webpack-cli migrate <config>
+```
+
+Given a basic configuration file like so:
+
+```javascript
```

Comment:
The link is provided at the start of this doc, which is essentially lifted right from the README

> The `migrate` feature eases the transition from [version 1](http://webpack.github.io/docs/) to [version 2](https://gist.github.com/sokra/27b24881210b56bbaff7). `migrate` 
also allows users to switch to the new version of webpack without having to extensively [refactor](https://webpack.js.org/guides/migrating/).

It's the link applied to **_refactor_**

Let me know if you feel it would be helpful to add it elsewhere as well.